docs-source/content/samples/simple/elastic-stack/weblogic-domain/_index.md

```diff
@@ -0,0 +1,340 @@
+The `fluentd` container will be configured to look for Elasticsearch parameters in the domain credentials.  Edit the domain credentials and add the parameters shown in the example below.
+
+Example: `kubectl edit secret bobs-bookstore-weblogic-credentials -n bob` and add the base64 encoded values of each Elasticsearch parameter:
+```text
+elasticsearchhost: ZWxhc3RpY3NlYXJjaC5ib2JzLWJvb2tzLnNhbXBsZS5jb20=
+elasticsearchport: NDQz
+elasticsearchuser: Ym9i
+elasticsearchpassword: d2VsY29tZTE=
+```
+
+## Create FluentD configuration
+Create a `ConfigMap` named `fluentd-config` in the namespace of the domain.  The `ConfigMap` contains the parsing rules and Elasticsearch configuration.
+
+An example of how to create the `ConfigMap`:
+```bash
+cat <<EOF | kubectl apply -f -
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  labels:
+    weblogic.domainUID: bobs-bookstore
+    weblogic.resourceVersion: domain-v2
+  name: fluentd-config
+  namespace: bob
+data:
+  fluentd.conf: |
+    <match fluent.**>
+      @type null
+    </match>
+    <source>
+      @type tail
+      path "#{ENV['LOG_PATH']}"
+      pos_file /tmp/server.log.pos
+      read_from_head true
+      tag "#{ENV['DOMAIN_UID']}"
+      # multiline_flush_interval 20s
+      <parse>
+        @type multiline
+        format_firstline /^####/
+        format1 /^####<(?<timestamp>(.*?))>/
+        format2 / <(?<level>(.*?))>/
+        format3 / <(?<subSystem>(.*?))>/
+        format4 / <(?<serverName>(.*?))>/
+        format5 / <(?<serverName2>(.*?))>/
+        format6 / <(?<threadName>(.*?))>/
+        format7 / <(?<info1>(.*?))>/
+        format8 / <(?<info2>(.*?))>/
+        format9 / <(?<info3>(.*?))>/
+        format10 / <(?<sequenceNumber>(.*?))>/
+        format11 / <(?<severity>(.*?))>/
+        format12 / <(?<messageID>(.*?))>/
+        format13 / <(?<message>(.*?))>/
+      </parse>
+    </source>
+    <match **>
+      @type elasticsearch
+      host "#{ENV['ELASTICSEARCH_HOST']}"
+      port "#{ENV['ELASTICSEARCH_PORT']}"
+      user "#{ENV['ELASTICSEARCH_USER']}"
+      password "#{ENV['ELASTICSEARCH_PASSWORD']}"
+      index_name "#{ENV['DOMAIN_UID']}"
+      scheme https
+      ssl_version TLSv1_2
+    </match>
+EOF
+```
+
+#### Mount the ConfigMap as a volume in weblogic-server container
+Edit the domain definition and configure a volume for the `ConfigMap` containing the `fluentd` configuration.
+
+NOTE: For brevity only the paths to the relevant configuration being added is shown.  A complete example of a domain definition is at the end of this document.
+
+Example: `kubectl edit domain bobs-bookstore -n bob` and add the following portions to the domain definition.  
+```yaml
+spec:
+  serverPod:
+    volumes:
+    - configMap:
+        defaultMode: 420
+        name: fluentd-config
+      name: fluentd-config-volume
+```
+
+#### Add fluentd container
+Add a container to the domain that will run `fluentd` in the admin and managed server pods.
```

Comment:
admin and managed server -> Administration Server and Managed Server (these WLS terms are always capitalized)